lib/handlers/quote/quote.ts

```diff
@@ -153,10 +156,14 @@ export class QuoteHandler extends APIGLambdaHandler<
         recipient: recipient,
         slippageTolerance: slippageTolerancePercent,
       }
+    } else {
+      // Can't simulate without swapParams set
+      simulate = false
     }
```

Comment:
Should just update the Joi validation to make the request fail if simulate is set, but swap params aren't